earthmover/earthmover.py

```diff
@@ -113,7 +113,7 @@ def load_config_file(self) -> dict:
                     continue
 
                 # Find the end of the config block (i.e., the next top-level field)
-                if start is not None and not line.startswith(tuple(string.whitespace)):
+                if start is not None and not line.startswith(tuple(string.whitespace)+tuple("#")):

```

Comment:
Just an FYI, we can change the second half of this line to the following:
`line.startswith(tuple(string.whitespace+"#"))`

Either way will work.